opentargets.json

```
@@ -1967,6 +1988,16 @@
         "Viral Protein Synthesis"
       ]
     },
+    "releaseDate": {
+      "type": "string",
+      "description": "Date of the release of the data in a 'YYYY-MM-DD' format",
+      "pattern": "^\\d{4}-\\d{2}-\\d{2}$"
```

```suggestion
      "pattern": "(((19|20)([2468][048]|[13579][26]|0[48])|2000)[-]02[-]29|((19|20)[0-9]{2}[-](0[469]|11)[-](0[1-9]|[12][0-9]|30)|(19|20)[0-9]{2}[-](0[13578]|1[02])[-](0[1-9]|[12][0-9]|3[01])|(19|20)[0-9]{2}[-]02[-](0[1-9]|1[0-9]|2[0-8])))"
```

Use the same pattern as `studyStartDate` to avoid the -unlikely- scenario where the data is not valid (0000-05-06 or 2021-13-01)

---

Yes, this makes sense.